recipes/r-transformer/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-transformer
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/SantanderMetGroup/transformeR/archive/refs/tags/v2.1.0.tar.gz
+  sha256: 68502ad405e7d92b728271162e73a195c6b69454fc0be94a069101e608830adf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=3.5,<4
```

Comment:
This results into R 3.6 builds being done, which is not really supported by conda-forge anymore. Is that intended? Can be lifted also after the merge to get supported R 4 and R 4.1 builds.

Reply:
Yes that is the intention. The build for >=4 would work but at the moment it's not been supported. 

As I mentioned above, that some research artifacts that depends on this.